scripts/get-e2e-db-schema.ts

```diff
@@ -26,24 +28,36 @@ const connectionStringMapping: Record<ComboCase, string> = {
 const args = arg({
   '--os': String,
   '--node-version': String,
+  '--prisma-client-version': String,
   '--github-env': String,
 })
 
-const comboCase = parseComboCase(args['--node-version'] ?? '', args['--os'] ?? '')
+const schemaName = parseComboCase(
+  args['--node-version'] ?? '',
+  args['--os'] ?? '',
+  args['--prisma-client-version'] ?? ''
+)
 
 if (args['--github-env']) {
-  fs.append(args['--github-env'], `E2E_DB_SCHEMA=${connectionStringMapping[comboCase]}`)
+  fs.append(args['--github-env'], `E2E_DB_SCHEMA=${schemaName}`)
 } else {
-  process.stdout.write(connectionStringMapping[comboCase])
+  process.stdout.write(schemaName)
 }
 
 //
 // Helpers
 //
 
-function parseComboCase(nodeVersionInput: string, osInput: string): ComboCase {
+function parseComboCase(nodeVersionInput: string, osInput: string, prismaClientInput: string): string {
   const nodeVersion = nodeVersionParser.parse(nodeVersionInput)
   const os = osParser.parse(osInput)
   // eslint-disable-next-line
-  return [nodeVersion, os].join(' + ') as any
+  const comboCase = [nodeVersion, os].join(' + ') as ComboCase
+  const schema = connectionStringMapping[comboCase]
+  if (!prismaClientInput) {
+    return schema
+  } else {
+    const prismaClient = prismaClientParser.parse(prismaClientInput)
+    return schema + prismaClient
```

Comment:
```suggestion
    return [schema,prismaClient].join('_') 
```

Would be a more consistent format.